setup.py

```diff
@@ -38,7 +38,7 @@ def get_description():
     packages=find_packages(),
     # Dependencies
     install_requires=[
-        "Django >= 2.2, < 3.2",
+        "Django >= 2.2, <= 4.2",
         "django-email-utils >= 1.0, < 2.0",
         # DRF 3.10 is the first to support our minimum Django version of 2.2.
         "djangorestframework >= 3.10",
```

Comment:
DRF 3.14 is the first version to support Django 3.2, 4.0, and 4.1.

```suggestion
        "djangorestframework >= 3.14",
```